opencensus-shim/src/main/java/io/opentelemetry/opencensusshim/SpanConverter.java

```diff
@@ -0,0 +1,261 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.opencensusshim;
+
+import io.opencensus.common.Function;
+import io.opencensus.implcore.trace.RecordEventsSpanImpl;
+import io.opencensus.trace.Annotation;
+import io.opencensus.trace.AttributeValue;
+import io.opencensus.trace.EndSpanOptions;
+import io.opencensus.trace.Link;
+import io.opencensus.trace.MessageEvent;
+import io.opencensus.trace.Span;
+import io.opencensus.trace.SpanContext;
+import io.opencensus.trace.TraceOptions;
+import io.opencensus.trace.Tracestate;
+import io.opencensus.trace.export.SpanData;
+import io.opencensus.trace.export.SpanData.TimedEvent;
+import io.opentelemetry.api.OpenTelemetry;
+import io.opentelemetry.api.common.AttributeKey;
+import io.opentelemetry.api.common.Attributes;
+import io.opentelemetry.api.trace.Span.Kind;
+import io.opentelemetry.api.trace.SpanId;
+import io.opentelemetry.api.trace.TraceFlags;
+import io.opentelemetry.api.trace.TraceId;
+import io.opentelemetry.api.trace.TraceState;
+import io.opentelemetry.api.trace.Tracer;
+import java.util.EnumSet;
+import java.util.List;
+import java.util.Map;
+import java.util.concurrent.TimeUnit;
+
+class SpanConverter {
+
+  /**
+   * FakeSpan is used to represent OpenTelemetry spans in the OpenCensus context. Only the trace ID,
+   * span ID, trace options, and trace state are mapped, so that the correct context information can
+   * be picked up by the child spans. FakeSpan does not record events or links.
```

Comment:
Doesn't it record events because we pass the RECORD_EVENTS option?

Reply:
What I'm trying to say with the comment is that the FakeSpan doesn't record events since `.addAnnotation()` is never called on it, and that it doesn't do anything even if `.addAnnotation()` was called on it, but it is the OpenCensus representation of the OpenTelemetry span which does record events. It has the record events option, which according to the `RECORD_EVENTS` documentation means "This option is set if the Span is part of a sampled distributed trace", so having this option just means it's sampled.

Reply:
Let me know if that doesn't make sense, happy to modify the comment :)

Reply:
Ah got it - since we have a word event with a specific meaning it was confusing. "Does not record" by itself seems fine to me.